browser/src/core/Socket.js

```diff
@@ -576,6 +577,35 @@ app.definitions.Socket = L.Class.extend({
 					setTimeout(reloadFunc, 5000);
 				}
 			}
+			if (window.indirectSocket) {
+				if (window.expectedServerId != this.WSDServer.Id) {
+					if (this.IndirectSocketReconnectCount++ >= 3) {
+						this._map.uiManager.showInfoModal('wrong-server-modal', _('Cluster configuration warning'), _('Collaborative editing or copy/paste might not work correctly, there are chances of potential document split'), '', _('OK'), null, false);
```

Comment:
"Your %product% cluster appear to be mis-configured or scaling rapidly - please contact your system administrator. Continuing with editing may result in multiple users not seeing each other, conflicts in the document storage and/or copy/paste problems."

Reply:
done, thanks